### ISSUE TITLE
Mirador can't correctly redraw complex annotations

### ISSUE THREAD

**Issue by dmnk1220** [ProjectMirador/mirador#1517]
When we add some complex annotations in the same page (e.g. overlapping regions in 2 annotations): all annotations in this page disappear and the page is "burnt" (we cannot add annotations anymore)
The following example is available as gif at:
[https://drive.google.com/open?id=1zcaaHVPaQ5-g9ncbO6Oaetk3JurqEIyR](https://drive.google.com/open?id=1zcaaHVPaQ5-g9ncbO6Oaetk3JurqEIyR)
and we described it step by step as follows:

Codex: Reg.lat. 708

1. We open the manuscript at page 3v - this is a page "burnt" because we cannot  display neither add annotations anymore.
2. We try to click on the toggle tool in order to display the annotations previously added (some complex annotation inside) but nothing happens
3. We jump to the next page, 4r in which we display some annotations (no issues for them).
4. We go back to the previous page 3v and, on this page, there are now the yellow shapes coming from the annotations pertaining to 4r page but we cannot display the content of these annotations.
5. Now we go back to page 3r (no annotations inside) and then  we go back again to 3v page and the  empty shapes disappear
6. We go back again to 4r page were annotations are correctly displayed and again we go back to the page 3v and again we found the empty shapes pertaining to the annotations of page 4r

   In a nutshell: 3v page doesn't display its annotations and takes the empty shapes from the next page which has some annotations.

We suspect it happens (to confirm) in case of complex annotations added...but of course we need to have enclaves of annotations so we need to fix this crucial issue.

Thank you in advance for your help.

Mr. Domenico Izzo
IT department Vatican Library

**Comment by blalbrit** [comment]
Thank you for this detailed description @dmnk1220 ! I can confirm the problem and see that there is an error being thrown from the Simple Annotation Server endpoint (https://github.com/ProjectMirador/mirador/blob/develop/js/src/annotations/simpleASEndpoint.js#L99) with a generic error message: 
"The request for annotations has caused an error for endpoint: https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012 due to error"

@glenrobson and @aeschylus  - have you seen this behavior before?

**Comment by blalbrit** [comment]
Looking a bit further up the console, there is also: 
`Failed to load {server omitted}/simpleAnnotationStore/annotation/search?uri=https%3A%2F%2Fdigi.vatlib.it%2Fiiif%2FMSS_Reg.lat.708%2Fcanvas%2Fp0012&APIKey=user_auth&media=image&limit=10000&_=1517936863629: No 'Access-Control-Allow-Origin' header is present on the requested resource. Origin '{server omitted' is therefore not allowed access.`

@eefahy - does this ring a bell at all?

**Comment by eefahy** [comment]
It would definitely make sense that annotations made on a canvas that does not have CORS enabled could have the effect that @dmnk1220 describes. The test would be to correct the CORS configuration for all canvases within the manifests and attempt to make the annotation again.

**Comment by glenrobson** [comment]
Hi,

Would it be possible to attach or send the output if you go to the following page:

```{server omitted}/simpleAnnotationStore/annotation/search?uri=https%3A%2F%2Fdigi.vatlib.it%2Fiiif%2FMSS_Reg.lat.708%2Fcanvas%2Fp0012&APIKey={omitted}&media=image&limit=10000&_=1517936863629```

(with omitted parts not omitted :-)). I suspect there is an annotation on that page that the SimpleAnnoationServer (SAS) can't process. I think when it throws a 500 error it doesn't include the CORS headers which is causing the CORS issue. 

Regarding the cause of this issue, so it happens when you have two seperate annotations where the boxes cross over each other, is that right?

**Comment by dmnk1220** [comment]
Hi,

regard this request:

**> Would it be possible to attach or send the output if you go to the following page:
> 
> {server omitted}/simpleAnnotationStore/annotation/search?uri=https%3A%2F%2Fdigi.vatlib.it%2Fiiif%2FMSS_Reg.lat.708%2Fcanvas%2Fp0012&APIKey={omitted}&media=image&limit=10000&_=1517936863629**

here is the output with the omitted parts non omitted:

[ {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515514632472",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:17:12",
  "resource" : [ {
    "@id" : "_:b0",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Il segno d&rsquo;interpunzione in forma di punto e virgola &egrave; aggiunto da una mano posteriore.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Il segno d&rsquo;interpunzione in forma di punto e virgola &egrave; aggiunto da una mano posteriore.</p>"
  }, {
    "@id" : "_:b3",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  }, {
    "@id" : "_:b4",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "Segni di interpunzione",
    "chars" : "Segni di interpunzione"
  } ],
  "on" : {
    "@id" : "_:b1",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b2",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b5",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=1562,526,27,52"
      },
      "item" : {
        "@id" : "_:b6",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M1565.8387,533.08217c2.483,-4.3305 5.98567,-7.55528 9.50531,-7.55142v0v0c3.51964,0.00385 6.58585,3.02058 9.50531,7.55142c2.91947,4.53084 4.05423,11.43498 3.93723,18.23075c-0.117,6.79577 -1.48575,13.48319 -3.93723,18.23075c-2.45147,4.74756 -5.98567,7.55528 -9.50531,7.55142c-3.51964,-0.00385 -7.02474,-2.81927 -9.50531,-7.55142c-2.48057,-4.73215 -3.93662,-11.38104 -3.93723,-18.23075c-0.00061,-6.84971 1.45423,-13.90025 3.93723,-18.23075z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"ellipse_bf24c025-fc74-40e0-acbf-a14872794a2a\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515515957034",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:39:17",
  "resource" : [ {
    "@id" : "_:b5",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Lo scriba aveva forse iniziato a tracciare un&rsquo;altra lettera dove poi ha scritto -m, cui segue uno spazio bianco della lunghezza di circa tre lettere.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Lo scriba aveva forse iniziato a tracciare un&rsquo;altra lettera dove poi ha scritto <em>-m</em>, cui segue uno spazio bianco della lunghezza di circa tre lettere.</p>"
  }, {
    "@id" : "_:b0",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  } ],
  "on" : {
    "@id" : "_:b3",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b4",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b1",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=505,1606,181,57"
      },
      "item" : {
        "@id" : "_:b2",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M504.73215,1605.90607h90.29675v0h90.29675v28.43454v28.43454h-90.29675h-90.29675v-28.43454z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_4f8fa8c6-76df-4971-a944-b257222d34bd\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515515646114",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:34:06",
  "resource" : [ {
    "@id" : "_:b5",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Nell&rsquo;interlineo un altro lettore, tre o quattrocentesco, ha scritto con inchiostro pi&ugrave; scuro bardelle.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Nell&rsquo;interlineo un altro lettore, tre o quattrocentesco, ha scritto con inchiostro pi&ugrave; scuro <em>bardelle.</em></p>"
  }, {
    "@id" : "_:b0",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  } ],
  "on" : {
    "@id" : "_:b1",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b2",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b3",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=1068,894,157,45"
      },
      "item" : {
        "@id" : "_:b4",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M1067.63831,894.05564h78.46224v0h78.46224v22.51729v22.51729h-78.46224h-78.46224v-22.51729z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_821f6b5d-74c0-4bee-816b-9798a175c85b\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515516046935",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:40:46",
  "resource" : [ {
    "@id" : "_:b3",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Dove ha scritto a di amorem&nbsp;, lo scriba aveva in un primo momento principiato una lettera diversa.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Dove ha scritto <em>a</em> di <em>amorem</em>&nbsp;, lo scriba aveva in un primo momento principiato una lettera diversa.</p>"
  }, {
    "@id" : "_:b4",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  } ],
  "on" : {
    "@id" : "_:b1",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b2",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b5",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=1212,1598,161,59"
      },
      "item" : {
        "@id" : "_:b0",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M1211.57507,1598.37502h80.61397v0h80.61397v29.51041v29.51041h-80.61397h-80.61397v-29.51041z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_2f9ba29f-ab35-4034-8ba5-4bd6904f340f\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515514526145",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:15:26",
  "resource" : [ {
    "@id" : "_:b0",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Nel codice niobis per il particolare modo di tracciare la n che contraddistingue talvolta questo scriba.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Nel codice <em>niobis</em> per il particolare modo di tracciare la <em>n</em> che contraddistingue talvolta questo scriba.</p>"
  }, {
    "@id" : "_:b5",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  } ],
  "on" : {
    "@id" : "_:b4",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b3",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b1",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=685,340,88,49"
      },
      "item" : {
        "@id" : "_:b2",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M684.82348,340.27094h44.18182v0h44.18182v24.63636v24.63636h-44.18182h-44.18182v-24.63636z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_95bdcb88-e345-4bc3-b25f-0013202da551\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1515514688601",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-09T17:18:08",
  "resource" : [ {
    "@id" : "_:b5",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Cos&igrave; nel codice.",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Cos&igrave; nel codice.</p>"
  }, {
    "@id" : "_:b4",
    "@type" : "oa:Tag",
    "http://dev.llgc.org.uk/sas/full_text" : "A. Berloco (a cura di)",
    "chars" : "A. Berloco (a cura di)"
  } ],
  "on" : {
    "@id" : "_:b0",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b1",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b2",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=699,618,93,60"
      },
      "item" : {
        "@id" : "_:b3",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M699.36894,617.54367h46.45455v0h46.45455v30.09091v30.09091h-46.45455h-46.45455v-30.09091z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_33dfc8a8-635d-47b1-b483-21c5fb78d68e\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting", "oa:tagging" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
}, {
  "@id" : "http://{server omitted}/simpleAnnotationStore/annotation/1516198971266",
  "@type" : "oa:Annotation",
  "dcterms:created" : "2018-01-17T15:22:51",
  "resource" : [ {
    "@id" : "_:b4",
    "@type" : "dctypes:Text",
    "http://dev.llgc.org.uk/sas/full_text" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Una mano diversa da quella del copista ha aggiunto sopra f una h. La stessa mano:\n<p class=\"MsoListParagraphCxSpFirst\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol; mso-bidi-font-style: italic;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; al r. 10&nbsp; corregge in mandavit espungendo la -b- e sovrascrivendole v; \n<p class=\"MsoListParagraphCxSpMiddle\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; al r. 15 aggiunge h nell&rsquo;interlineo prima di istoria e&nbsp; corregge in prophetiis aggiungendo una seconda -i- nell&rsquo;interlineo\n<p class=\"MsoListParagraphCxSpMiddle\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol; mso-bidi-font-style: italic;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; al r . 22 corregge in debeamus tracciando una b direttamente sopra la &ndash;u\n&nbsp;\n<p class=\"MsoListParagraphCxSpLast\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; al r. 23 ha cos&igrave; corretto da precedente interpreto (?) aggiungendo un trattino in basso alla o e alle lettere ri",
    "format" : "text/html",
    "chars" : "<p class=\"MsoNormal\" style=\"margin-bottom: 10.0pt; text-align: justify; line-height: 115%;\">Una mano diversa da quella del copista ha aggiunto sopra <em>f</em> una <em>h</em>. La stessa mano:</p>\n<p class=\"MsoListParagraphCxSpFirst\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol; mso-bidi-font-style: italic;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>al r. 10&nbsp; corregge in <em>mandavit</em> espungendo la <em>-b-</em> e sovrascrivendole <em>v; </em></p>\n<p class=\"MsoListParagraphCxSpMiddle\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>al r. 15 aggiunge <em>h </em>nell&rsquo;interlineo prima di <em>istoria </em>e&nbsp; corregge in <em>prophetiis</em> aggiungendo una seconda <em>-i-</em> nell&rsquo;interlineo</p>\n<p class=\"MsoListParagraphCxSpMiddle\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol; mso-bidi-font-style: italic;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>al r . 22 corregge in <em>debeamus</em> tracciando una <em>b</em> direttamente sopra la <em>&ndash;u</em></p>\n<p>&nbsp;</p>\n<p class=\"MsoListParagraphCxSpLast\" style=\"margin-bottom: 10.0pt; mso-add-space: auto; text-align: justify; text-indent: -18.0pt; line-height: 115%; mso-list: l0 level1 lfo1;\"><span style=\"font-family: Symbol; mso-fareast-font-family: Symbol; mso-bidi-font-family: Symbol;\">&middot;<span style=\"font-variant-numeric: normal; font-variant-east-asian: normal; font-stretch: normal; font-size: 7pt; line-height: normal; font-family: 'Times New Roman';\">&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp; </span></span>al r. 23 ha cos&igrave; corretto da precedente <em>interpreto</em> (?) aggiungendo un trattino in basso alla <em>o</em> e alle lettere <em>ri</em></p>"
  } ],
  "on" : {
    "@id" : "_:b3",
    "@type" : "oa:SpecificResource",
    "within" : {
      "@id" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/manifest.json",
      "@type" : "sc:Manifest"
    },
    "selector" : {
      "@id" : "_:b1",
      "@type" : "oa:Choice",
      "default" : {
        "@id" : "_:b0",
        "@type" : "oa:FragmentSelector",
        "value" : "xywh=509,286,246,87"
      },
      "item" : {
        "@id" : "_:b2",
        "@type" : "oa:SvgSelector",
        "value" : "<svg xmlns='http://www.w3.org/2000/svg'><path xmlns=\"http://www.w3.org/2000/svg\" d=\"M509.40171,285.52h122.85371v0h122.85371v43.44343v43.44343h-122.85371h-122.85371v-43.44343z\" data-paper-data=\"{&quot;strokeWidth&quot;:3,&quot;rotation&quot;:0,&quot;deleteIcon&quot;:null,&quot;rotationIcon&quot;:null,&quot;group&quot;:null,&quot;editable&quot;:true,&quot;annotation&quot;:null}\" id=\"rectangle_d07bb7b0-caa4-468d-adc2-1b3e3ac0b1c3\" fill-opacity=\"0\" fill=\"#00bfff\" fill-rule=\"nonzero\" stroke=\"#ffff00\" stroke-width=\"3\" stroke-linecap=\"butt\" stroke-linejoin=\"miter\" stroke-miterlimit=\"10\" stroke-dasharray=\"\" stroke-dashoffset=\"0\" font-family=\"none\" font-weight=\"none\" font-size=\"none\" text-anchor=\"none\" style=\"mix-blend-mode: normal\"/></svg>"
      }
    },
    "full" : "https://digi.vatlib.it/iiif/MSS_Reg.lat.708/canvas/p0012"
  },
  "motivation" : [ "oa:commenting" ],
  "@context" : "http://iiif.io/api/presentation/2/context.json"
} ]

For the question:

> **Regarding the cause of this issue, so it happens when you have two seperate annotations where the boxes cross over each other, is that right?**

I confirmed that the issue pointed out in MSS_Reg.lat.708/canvas/p0012
is pertaining an intersection (let's say a "complex annotation") but since the beginning of production in Mirador (started at the BAV in December 2017) we had serveral pages "burnt": we are able to report each annotation failed so tomorrow we will give you the link in which you will fin the description of each one and also the error messages (if available) from Solr and SAS.

**Comment by blalbrit** [comment]
@glenrobson - does any of this ^^ ring a bell?

**Comment by dmnk1220** [comment]
This issue has been resolved. I am grateful to the team that worked to solve it by replacing the SAS endpoint.
